src/chain_spec.rs

```diff
@@ -42,6 +39,55 @@ pub type ChainSpec = sc_service::ChainSpec<
 	Extensions,
 >;
 
+/// The chain specification option.
+#[derive(Clone, Debug, PartialEq)]
+pub enum Alternative {
+	/// Whatever the current runtime is, with just Alice as an auth.
+	Development,
+	/// Whatever the current runtime is, with simple Alice/Bob auths.
+	LocalTestnet,
+	/// The Fulvous testnet.
+	Fulvous,
+	/// The Flint testnet.
+	Flint,
+	/// The Amber testnet.
+	Amber,
+}
+
+/// Get a chain config from a spec setting.
+impl Alternative {
+	pub(crate) fn load(self) -> Result<ChainSpec, String> {
+		Ok(match self {
+			Alternative::Development => development_config(),
+			Alternative::LocalTestnet => local_testnet_config(),
+			Alternative::Fulvous => fulvous_config(),
+			Alternative::Flint => flint_config()?,
+			Alternative::Amber => amber_config()?,
+		})
+	}
+
+	pub(crate) fn from(s: &str) -> Option<Self> {
+		match s {
+			"dev" => Some(Alternative::Development),
+			"local" => Some(Alternative::LocalTestnet),
+			"fulvous" => Some(Alternative::Fulvous),
+			"flint" => Some(Alternative::Flint),
+			"" | "amber" => Some(Alternative::Amber),
```

Comment:
I think being explicit for amber would be good. Having an empty `""` would be good chioce for mainnet Chainspc

Reply:
Agree, will change.